app/utils/ControlProportionSettings.scala

```diff
@@ -0,0 +1,3 @@
+package utils
```

Comment:
👍 for a new file, but this can live in the `models` package

Reply:
Done! ✅